compaction/compaction_strategy.cc

```diff
@@ -120,10 +120,10 @@ size_tiered_backlog_tracker::compacted_backlog(const compaction_backlog_tracker:
     return in;
 }
 
-void size_tiered_backlog_tracker::refresh_sstables_backlog_contribution() {
+size_tiered_backlog_tracker::contribution size_tiered_backlog_tracker::calculate_sstables_backlog_contribution(const std::unordered_set<sstables::shared_sstable>& all, const sstables::size_tiered_compaction_strategy_options& stcs_options) {
```

Comment:
Should it be const?

Perhaps even static (and receive parameters). "calculate" is an indication that it wants to be a pure function.

Reply:
dropped in v2